app/lib/pyramidal_tiff_factory.rb

```diff
@@ -77,13 +77,13 @@ def copy_access_master_to_working_directory(tmpdir)
   # We don't know for sure where the access master mount will be, this is a default for local development for now
   def self.access_master_path(oid)
     image_mount = ENV['ACCESS_MASTER_MOUNT'] || "data"
-    pairtree_path = Partridge::Pairtree.oid_to_pairtree(oid)
+    pairtree_path = Partridge::Pairtree.oid_to_pairtree(oid) || ''
```

Comment:
This was because the pairtree returns nil on oids with one digit, and the tests used an oid like that.
It would probably be better to update the gem to return blank or some thing when it gets an short oid...this is really a work around to get the test to pass with a single digit oid without updating the gem.